matrix-room.c

```diff
@@ -865,6 +865,49 @@ void matrix_room_handle_timeline_event(PurpleConversation *conv,
     g_free(escaped_body);
 }
 
+/**
+ * Callback for after the room is created to tweak it to publically accessible
```

Comment:
you should be able to do that in one step. set `"visibility": "public"` in the body of the `createRoom` request.